### PR TITLE
SEQNG-195: Automatic load and unload of sequences from ODB queue.

### PR DESCRIPTION
**Pull request by jluhrs**
The mechanism implemented is a periodic polling. Every 10 seconds Seqexec retrieves the list of sequences in the ODB queue and compares it with the sequences loaded. New sequences in the list will be loaded, and sequences not in the list will be unloaded.

### CODE REVIEW

File: modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala [hunk 1]
```diff
@@ -76,13 +79,14 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
   def setCloudCover(q: engine.EventQueue, cc: CloudCover): Task[SeqexecFailure \/ Unit] =
     q.enqueueOne(Event.setCloudCover(cc)).map(_.right)
 
-  // TODO: Add seqId: SPObservationID as parameter
   def setSkipMark(q: engine.EventQueue, id: SPObservationID, stepId: edu.gemini.seqexec.engine.Step.Id): Task[SeqexecFailure \/ Unit] = ???
 
   def requestRefresh(q: engine.EventQueue): Task[Unit] = q.enqueueOne(Event.poll)
 
+  def seqQueueRefreshProcess(q: engine.EventQueue): Process[Task, Event]= awakeEvery(10.seconds)(Strategy.DefaultStrategy, DefaultScheduler).map(_ => Event.getState(refreshSequenceList(q)))
```
Comment (cquiroz) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala]: Should the time be configurable?

File: modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala [hunk 2]
```diff
@@ -192,6 +202,14 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
     // TODO: Implement willStopIn
     SequenceView(seq.id, seq.metadata, st, engineSteps(seq), None)
   }
+
+  private def refreshSequenceList(q: engine.EventQueue): Engine.State => Task[Unit] = {
+    st: Engine.State => odbProxy.queuedSequences().flatMap(seqs => EitherT(Nondeterminism[Task].gatherUnordered{
```
Comment (cquiroz) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala]: This is quite hard to follow, perhaps it could be split for better readability? Could you run this inside a for comprehension?

File: modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala (hunk 1 above)
Comment (jluhrs) [inline modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala]: I forgot to do that. I'll put it right now.
